resolvers/__tests__/index.spec.js

```diff
@@ -62,6 +62,13 @@ beforeAll(async () => {
   stubs.userActions = [userActionOne, userActionTwo]
 })
 
+afterAll(async () => {
```

Comment:
We can refactor this later, but this should be done `afterEach` instead. Every test should be isolated from others, and having the database clean before each test is key to achieve it